src/card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  width: 200px;
+
+  background: var(--bg-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.card__info {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-top: 40px;
+  padding-bottom: 16px;
+}
+
+.card__info-title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  margin: 0;
+}
+
+.card__info-code {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary-text-color);
+
+  margin: 0;
+  padding-top: 4px;
+  padding-bottom: 16px;
+}
+
+.card__info-reviews {
+  display: flex;
+  justify-content: space-between;
+
+  font-size: 10px;
+  line-height: 14px;
```

Comment:
I think you can remove this line, it's completely no sense. I guess you just copied it from figma)

Reply:
Yes I copied it from figma, and it's not working without this line